File: Manajero-ASD-Sprint2.2-3/src/app/pages/asd/Entities/Backlog/TaskDTO.ts
import { Priority } from "./priority.enum";

export interface TaskDTO {
    taskName: string;
    description?: string;
    dueDate: string;
    priority:Priority;
    isCompleted: boolean;
    sprintTo: string; // Assuming this is the ID or some identifier for the Iteration
  }

  

File: Manajro-ASD-BackEnd-Sprint2.2-3/src/main/java/com/manajero/backend_springboot/dto/TaskDTO.java
package com.manajero.backend_springboot.dto;

import com.manajero.backend_springboot.collections.Backlog.Priority;
import lombok.*;

import java.time.LocalDateTime;
import org.springframework.data.annotation.Id;
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@ToString
public class TaskDTO {
    @Id
    private String taskId;
    private String taskName;
    private String description;
  ////  private LocalDateTime createdDate;
  private Priority priority;
    private LocalDateTime dueDate;
    private String sprintTo;  // String to hold the Iteration ID

    private Boolean isCompleted;


}
